usbtree.c

```diff
@@ -251,13 +252,16 @@ static void DisplayDevice (struct Device *parent, struct Device *device)
 	}
 
 	/* change the color of this leaf if there are no drivers attached to it */
-	if (driverAttached == FALSE)
+	if (driverAttached == FALSE) {
 		color = "red";
+		tooltip = "This device has no attached driver";
+	}
 
 	gtk_tree_store_set (treeStore, &device->leaf,
 			    NAME_COLUMN, device->name,
 			    DEVICE_ADDR_COLUMN, deviceAddr,
 			    COLOR_COLUMN, color,
+				TOOLTIP_COLUMN, tooltip,
```

Comment:
Oh, sure, the one spot I actually remember to indent with tabs...

```suggestion
			    TOOLTIP_COLUMN, tooltip,
```